### ISSUE TITLE
Mod request: Dynamic FPS

### ISSUE THREAD

**Issue by MenacingPerson** [intergrav/additive#22]
Dynamic FPS automatically reduces the speed at which Minecraft renders when it's not focused (to 1 FPS) or hidden (no renders at all).

It also fixes a bug in Vanilla Minecraft that makes it take much more performance in the background than it should, as well as making the game not render while reloading resources (which can make it take forever on machines with integrated GPUs).

https://modrinth.com/mod/dynamic-fps

**Comment by MenacingPerson** [comment]
Really? I mean, it does improve performance when *tabbed out*.

Specifically performance of other apps. I would still consider it a performance mod TBH.

**Comment by Gaming32** [comment]
I agree. I would also consider it a performance mod for the same reason. Its goal is performance and optimizing Minecraft's idle state.
